docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Ubuntu/README.md

```diff
@@ -1 +1,165 @@
+| grep 'tarball' \
+| cut -d : -f2,3 \
+| tr -d \" \
+| tr -d , \
+| wget -i - -O loris-src.tar.gz
+```
+
+When this is complete, expand the compressed file and move its contents to the
+directory from which LORIS will be served. 
+
+```bash
+tar -zxf loris-src.tar.gz
+```
+
+This will create a folder called something like `aces-Loris-9e30cf0`. (The
+part after `aces-Loris` is not important).
+
+Rename this file to `loris`, e.g.
+
+```bash
+mv aces-Loris-9e30cf0 loris
+```
+
+Then move this folder to the web root (e.g. `/var/www/`) and go to this 
+directory.
+
+```bash
+mv ./loris /var/www/; cd /var/www/loris
+
+### If installing LORIS for production
+
+```bash
+cd `/var/www/loris/`; make
+```
+
+### If installing LORIS for development
+
+```bash
+# This is necessary for 'phan', a static analysis tool
+sudo pecl install ast-1.0.3
+cd `/var/www/loris/`; make dev
+```
+
+The `dev` target for `make` will install additional libraries that are needed 
+for development process but not for production installs.
+
+## Running the install script
+
+The next step in setting up LORIS is running the script `install.sh` in the 
+`tools/` directory. The script must be run from this directory.
+
+This will begin an interactive setup process that will configure files and
+permissions required to get LORIS up and running.
+
+The install script will prompt for the user password in order to use `sudo` permission for configuring
+certain file permissions as well as Apache. However, the script should not
+be run as `root` and will exit automatically if it detects that it has been
+executed with superuser permissions.
+
+```bash
+cd tools/
+./install.sh
+```
+
+## Configuring the database
+
+The install script will tell you to navigate to <loris-url>/installdb.php.
+
+MySQL (or MariaDB) must be installed and a root or admin-level MySQL user must
+be created before continuing. (This is not the same as a unix root credential.)
+
+This web page will prompt you for the following information:
+
+ * `Server Hostname`. Use `localhost` if your database is hosted on your locally. Use the IP address of your database server otherwise.
+
+ * `Admin Username` A database user with permission to create databases and tables.
+
+ * `Admin Password` The password for the above database user.
+
+ * `Database Name` Defaults to "LORIS".
+
+Click submit, and on the next screen that is presented, follow instructions to enter the username and password of your LORIS database user and front-end `admin` user.
+
+If you encounter issues creating/generating your config file, you may have to manually paste the xml output that appears on the screen into the file `/var/www/loris/project/config.xml`
+
+Your LORIS instance should now be accessible by pointing your browser URL to `http://%IPADDRESS%`.
+
+
+Now that the installation is complete, follow the [Setup process](https://github.com/aces/Loris/wiki/Setup) to customize your project.
+
+### Development
+
```

Comment:
Suggestion - Given there's a Development section here, we could/should mention setting the config.xml parameter here (and if not here, in the following Setup step), and anything you might do differently in dev/production.

Reply:
Since that information doesn't depend on the Operating System, I think it should be done in another document.